thoth/metrics_exporter/jobs/ceph.py

```diff
@@ -71,3 +44,13 @@ def get_ceph_connection_error_status() -> None:
             _LOGGER.exception(excptn)
         else:
             metrics.ceph_connection_error_status.set(1)
+
+
+def get_ceph_results_per_type(store: ResultStorageBase) -> None:
```

Comment:
I would probably move this directly to `MetricsBase` to have it available across all the child classes and have it accessible via `self`.

Reply:
Thanks, will do :) :coffe

Reply:
Updated in https://github.com/thoth-station/metrics-exporter/pull/383/commits/f9d2716e14c691593ef88062dc92ca8930744f8d